apps/coscrad-frontend-e2e/src/e2e/resources/terms/term.query-flow.e2e.cy.ts

```diff
@@ -27,13 +27,17 @@ describe(`Term index-to-detail flow`, () => {
         });
```

Comment:
I'm assuming the term tests are in good shape too. Let's be sure to keep an eye on them.